index.html

```diff
@@ -0,0 +1,289 @@
+                                <div class="main-news-image">
+                                        <a href="#" class="card-link">
+                                                <img src="./img/gemma-evans-alVcL70NHUU-unsplash.jpg" 
+                                                alt="Man carving a spoon" class="card-image">
+                                        </a>
+                                </div>
+                                <div class="main-news-textbox">
+                                        <h2 class="main-news-title">Spoon it!</h2>
+                                        <p class="main-news-text">
+                                                Bland herrskaperna på läktaren, idel fiender.
+                                        </p>
+                                </div>
+                        </article>
+
+                        <article class="main-news-item">
+                                <div class="main-news-image">
+                                        <a href="#" class="card-link">
+                                                <img src="./img/j-williams-dRRU9EbH5eQ-unsplash.jpg" 
+                                                alt="Fixing a seam on sewn item" class="card-image">
+                                        </a>
+                                </div>
+                                <div class="main-news-textbox">
+                                        <h2 class="main-news-title">Fixin' it.</h2>
+                                        <p class="main-news-text">
+                                                Prästen skulle ha velat störta ner på sina knän och be
+                                                dem om förbarmande.
+                                        </p>
+                                </div>
+                        </article>
+                </section>
+        </main>
+        <footer>
+                <div class="footer-info">
+                        <p class="footer-text">
+                                Evelina Berg 
+                                <br>A Technigo Bootcamp project
+                        </p>
+                        <div class="social-media-items">
+                                <a href="https://www.linkedin.com/in/evelina-berg-49540753/" class="social-media-link">
+                                        <i class="fab fa-linkedin-in fa-2x"></i>
+                                </a>
+                                <a href="https://github.com/evelinabe" class="social-media-link">
+                                        <i class="fab fa-github fa-2x"></i>
+                                </a>
+                                <a href="https://www.instagram.com/evelinabe_/" class="social-media-link">
+                                        <i class="fab fa-instagram fa-2x"></i>
+                                </a>
+                        </div>
+                </div>        
+        </footer>
```

Comment:
Really good looking footer, both style wise and how you've built it in HTML. So cool to see 🏆